app/models/binary_blob/purging.rb

```diff
@@ -0,0 +1,20 @@
+class BinaryBlob < ApplicationRecord
+  module Purging
+    extend ActiveSupport::Concern
+    include PurgingMixin
+
+    module ClassMethods
+      def purge_timer
+        purge_queue(:scope)
+      end
+
+      def purge_window_size
+        ::Settings.binary_blob.purge_window_size
+      end
+
+      def purge_scope(_older_than = nil)
+        where(:resource => nil)
```

Comment:
TODO see https://github.com/ManageIQ/manageiq/pull/16413/files#r151449186

Reply:
Let's get this in as part of this PR.